contracts/cairo/deploy.sh

```diff
@@ -7,27 +7,10 @@ PINK='\033[1;35m'
 ORANGE='\033[1;33m'
 COLOR_RESET='\033[0m'
 
-cd "$(dirname "$0")"
-
-if [ -f .env ]; then
-    echo "Sourcing cairo/.env file..."
-    source .env
-else
-    echo "Error: cairo/.env file not found!"
-    exit 1
-fi
-
-if ! grep -q "^SN_ESCROW_OWNER=" ".env"; then
-  echo "" #\n
-  echo -e "${ORANGE}WARNING:${COLOR_RESET} no SN_ESCROW_OWNER defined in .env, declaring deployer as the owner of the contract"
-  SN_ESCROW_OWNER=$(cat "$STARKNET_ACCOUNT" | grep '"address"' | sed -E 's/.*"address": "([^"]+)".*/\1/')
-fi
```

Comment:
This check is pretty important, it won't execute without it

Reply:
> There should be at least a check this .env var exists, if not, it will not deploy without much information
Do you mean "it will not fail without much information"?

Reply:
I meant, if SN_ESCROW_OWNER is not defined, deploy will fail without saying correctly why it has failed.
(I'm saying this because it happened to me quite a few times during dev)

Reply:
resolved